locales-pending/premium.ftl

```diff
@@ -395,3 +395,38 @@ security-recommendation-ip-summary = { $num_breaches ->
 }
 security-recommendation-ip-description = Your IP address pinpoints your location and internet service provider. Hackers could use this information to find your location or try to connect to your devices.
 security-recommendation-ip-step-one = Use a VPN (such as <link_to_info>{ -brand-mozilla-vpn }</link_to_info>) to hide your real IP address and use the internet privately.
+
+# Leaked Passwords
+
+# Variables
+# $breach_name is the name of the breach where the leaked password was found.
+leaked-passwords-title = Your { $breach_name } password was exposed.
+# Variables
+# $breach_date is the date when the breach occurred. 
+leaked-passwords-summary = It appeared in a data breach on { $breach_date }.
+leaked-passwords-description = Scammers can access your account and will likely try to use it on other accounts to see if you’ve used the same password. Change it anywhere you’ve used it to protect yourself.
+leaked-passwords-steps-title = Here’s what to do
+leaked-passwords-steps-subtitle = This requires access to your account, so you’ll need to manually fix it.
+# Variables
+# $breach_name is the name of the breach where the leaked password was found.
+leaked-passwords-step-one = Change your password on <link_to_breach_site>{ $breach_name }</link_to_breach_site>.
+leaked-passwords-step-two = Change it anywhere else youve used it.
+leaked-passwords-mark-as-fixed = Mark as fixed
+leaked-passwords-skip = Skip for now
+
+# Leaked Security Questions
+
+leaked-security-questions-title = Your security questions were exposed
+# Variables
+# $breach_name is the name of the breach where the leaked security questions were found.
+# $breach_date is the date when the breach occurred. 
+# An example of this string is Twitter on 13/09/18.
+leaked-security-questions-summary = They appeared in a data breach on { $breach_name } on { $breach_date }.
+leaked-security-questions-description = Scammers can use these to access your accounts, and any other site where you’ve used the same security questions. Update them now to protect your accounts.
+leaked-security-questions-steps-title = Here’s what to do
+leaked-security-questions-steps-subtitle = This requires access to your account, so you’ll need to manually fix it.
+# Variables
+# $breach_name is the name of the breach where the security questions were found.
+leaked-security-questions-step-one = Update your security questions on <link_to_breach_site>{ $breach_name }</link_to_breach_site>.
+leaked-security-questions-step-two = Update them on any other site where you used the same security questions. Be sure to use different security questions for every account.
```

Comment:
"you used" vs "you've used" for the leaked passwords. Is consistency important here?